corehq/pillows/sms.py

```diff
@@ -42,3 +56,36 @@ def calc_meta(cls):
         so we just do a hash of the "prototype" instead to determined md5
         """
         return cls.calc_mapping_hash({"es_meta": cls.es_meta, "mapping": cls.default_mapping})
+
+    def change_transport(self, doc_dict):
```

Comment:
@czue this is the right place in the couch SMSPillow to patch for the toggle right? i tested and it works but just want to doublecheck i didn't miss something